gitops/snapshot.go

```diff
@@ -324,6 +324,15 @@ func HaveAppStudioTestsFinished(snapshot *applicationapiv1alpha1.Snapshot) bool
 	return statusCondition != nil && statusCondition.Status != metav1.ConditionUnknown
 }
 
+func PrepareStartCreateRelease(snapshot *applicationapiv1alpha1.Snapshot) {
+	// Find the desired status condition
+	condition := meta.FindStatusCondition(snapshot.Status.Conditions, AppStudioTestSucceededCondition)
+
+	// Use the condition's LastTransitionTime timestamp as startCreateRelease timestamp
+	startCreateRelease := condition.LastTransitionTime
+	go metrics.RegisterReleaseLatency(startCreateRelease)
+}
```

Comment:
Two things here:

1. Where is `PrepareStartCreateRelease()` called?
2. I don't know that this needs to be its own function. I think you can take the two lines in here and just put them into `RegisterReleaseLatency()`

Reply:
suggested changes made, thank you.